INSTALL.md

```diff
@@ -67,18 +76,30 @@ make crystal11
 ```
 
 
-## Mac
+## Mac OS X
+
+Open **Terminal** and enter the following commands.
 
-In **Terminal**, run:
+### 1. Install the required software
+
+To install the **Xcode Command Line Tools**:
 
 ```bash
 xcode-select --install
+```
 
+To install **rgbds**:
+
+```bash
 git clone --depth=1 https://github.com/rednex/rgbds
-cd rgbds
-sudo make CFLAGS=-O2 install
-cd ..
+sudo make -C rgbds CFLAGS=-O2 install
```

Comment:
Can you confirm mac's version of make supports the -C option? I left it because I wan't entirely sure...